resources/translations/fi.edn

```diff
@@ -286,6 +286,6 @@
            :open-search "Avaa haku"
            :search-parameters "Hakutermit"}
   :unauthorized-page {:unauthorized "Ei käyttöoikeuksia"
-                      :you-are-unauthorized "Sinulla ei ole käyttöoikeuksia tähän sivuun."}
+                      :you-are-unauthorized "Ei sivun käyttöoikeuksia."}
```

Comment:
This should be matched to the heading. It can be shorter and the main explanation should be longer. See also English.

Reply:
Done.